e2e/miniconstellation/main.sh.in

```diff
@@ -28,6 +28,9 @@ terraform apply -auto-approve
 terraform output -raw ssh_private_key > id_rsa
 chmod 600 id_rsa
 
+rg_name=$(terraform output -raw rg_name)
+echo "rgname=$rg_name" >> "$GITHUB_OUTPUT"
```

Comment:
I think this will break usage of the test outside of GitHub CI.
Afaik only @3u13r ever really used this, so I would let him decide if we just tolerate this or if it should be changed

Reply:
I mean, changing this should be easy. GitHub sets a `CI` variable to `true` if a script is run from an action (https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) so we could just check for its existence and only append to `GITHUB_OUTPUT` if required.

Reply:
I think we can change the line to `echo "rgname=$rg_name" >> "${GITHUB_OUTPUT:-/dev/null}"`.

Reply:
I'm testing this now.

Reply:
> I think we can change the line to `echo "rgname=$rg_name" >> "${GITHUB_OUTPUT:-/dev/null}"`.

As far as I can see, this works ([example run that cleans up failed test](https://github.com/edgelesssys/constellation/actions/runs/9222624003/job/25374145602))